controller/label.go

```diff
@@ -149,3 +148,22 @@ func ConvertLabels(appl application.Application, request *http.Request, labels [
 	}
 	return ls
 }
+
+// ConvertLabelsSimple converts a array of Label IDs into a Generic Reletionship List
+func ConvertLabelsSimple(request *http.Request, labelIDs []interface{}) []*app.GenericData {
+	ops := []*app.GenericData{}
+	for _, labelID := range labelIDs {
+		ops = append(ops, ConvertLabelSimple(request, labelID))
+	}
+	return ops
+}
+
+// ConvertLabelSimple converts a Label ID into a Generic Reletionship
+func ConvertLabelSimple(request *http.Request, labelID interface{}) *app.GenericData {
+	t := label.APIStringTypeLabels
+	i := fmt.Sprint(labelID)
```

Comment:
A string assertion would be good enough here.

Reply:
yeah I checked that done in https://github.com/fabric8-services/fabric8-wit/pull/1627/commits/c016b4e144ce28e8661024f4f601e1b9d40fd668